Yoast/wordpress-seo
# Update rest api mailchimp settings

Sets the mailchimpActionUrl to the mailchimp jsonp endpoint and adds the username to the mailchimp signup field in the config-ui backend.

Testing:
1. Start the wizard, but the mailchimp component from:  #5511 has to be merged.
2. The first step shows the mailchimp component, fill in your own email.
3. Check if the subscription is succesfull and if you get an confirmation email for the Yoast mailing list. 

Fixes https://github.com/Yoast/yoast-components/issues/48
